From 3801eb47f3ce58d2f8b64089ab01d85edf19f441 Mon Sep 17 00:00:00 2001
From: warmcat <andy@warmcat.com>
Date: Sun, 13 Apr 2008 07:25:44 +0100
Subject: [PATCH] local-build-scripts.patch

Add a couple of handy scripts to crossmake and send over dfu

From: warmcat <andy@warmcat.com>
---
 build    |   14 ++++++++++++++
 dfu-kern |    9 +++++++++
 2 files changed, 23 insertions(+), 0 deletions(-)
 create mode 100755 build
 create mode 100755 dfu-kern

diff --git a/build b/build
new file mode 100755
index 0000000..7e61ea2
--- /dev/null
+++ b/build
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -x
+
+export CROSS_COMPILE=../../cross/bin/arm-angstrom-linux-gnueabi-
+make ARCH=arm silentoldconfig
+if make -j5 ARCH=arm; then
+	${CROSS_COMPILE}objcopy -O binary -R .note -R .comment -S arch/arm/boot/compressed/vmlinux linux.bin
+	mkimage -A arm -O linux -T kernel -C none -a 30008000 -e 30008000 -n "OpenMoko Kernel Image Neo1973(GTA02)" -d linux.bin uImage.bin
+	exit 0
+else
+	exit 1
+fi
+
diff --git a/dfu-kern b/dfu-kern
new file mode 100755
index 0000000..990abdd
--- /dev/null
+++ b/dfu-kern
@@ -0,0 +1,9 @@
+#!/bin/bash
+../../dfu-util/src/dfu-util -a 3 -d 0x1457:0x5119 -D uImage.bin
+if [ $? -eq 1 ] ; then
+../../dfu-util/src/dfu-util -a 3 -d 0x1457:0x5120 -D uImage.bin
+../../dfu-util/src/dfu-util -a 3 -d 0x1457:0x5119 -D uImage.bin
+
+fi
+
+
-- 
1.5.6.5

